test/integration/targets/cron/tasks/main.yml

```diff
@@ -0,0 +1,139 @@
+- include_vars: "{{ lookup('first_found', search) }}"
+  vars:
+    search:
+      files:
+        - 'cron.{{ ansible_system | lower }}.yml'
+        - 'cron.{{ ansible_distribution | lower }}.yml'
+        - 'cron.{{ ansible_os_family | lower }}.yml'
+      paths:
+        - '../defaults/'
+
+- vars:
+    remote_dir: "{{ lookup('env', 'OUTPUT_DIR') }}"
+  block:
+    - name: install cron package
+      package:
+        name: '{{ cron_pkg }}'
+      when: cron_pkg|default(false, true)
+      register: cron_package_installed
+      until: cron_package_installed is success
+
+    - when: faketime_pkg|default(false, true)
+      block:
+        - name: install cron and faketime packages
+          package:
+            name: '{{ faketime_pkg }}'
+          register: faketime_package_installed
+          until: faketime_package_installed is success
+
+        - name: Find libfaketime path
+          shell: '{{ list_pkg_files }} {{ faketime_pkg }} | grep -F libfaketime.so.1'
```

Comment:
Could this be done with [package_facts](https://docs.ansible.com/ansible/latest/modules/package_facts_module.html) instead? You might need the name of the package containing `libfaketime.so` for that, though.

Reply:
I don't know, the documentation of `package_facts`  only states that it returns `list of dicts with package information`.

Reply:
Yep, I also saw that. That's why I wrote it as a question ;-) I guess it returns a dict with keys corresponding to (installed?) packages, but that's all I know. I've tested it for Debian, there the result looks like:
```.yaml
{
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python",
        "packages": {
            "adduser": [
                {
                    "arch": "all",
                    "category": "admin",
                    "name": "adduser",
                    "origin": "Debian",
                    "source": "apt",
                    "version": "3.118"
                }
            ],
            "adwaita-icon-theme": [
                {
                    "arch": "all",
                    "category": "gnome",
                    "name": "adwaita-icon-theme",
                    "origin": "Debian",
                    "source": "apt",
                    "version": "3.30.1-1"
                }
            ],
            ...
        }
    }
}
```
This seems to be the list of actually installed packages. No idea what the result is for RHEL and others...

Reply:
I would prefer to not rely on something undocumented.